phpunit

```diff
@@ -41,7 +41,16 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit")) {
     $zip->close();
     chdir("phpunit-$PHPUNIT_VERSION");
     passthru("$COMPOSER remove --no-update symfony/yaml");
+    passthru("$COMPOSER require --no-update symfony/phpunit-bridge ~2.8");
```

Comment:
Well, phpunit listeners are loaded before the bootstrapp file is included, which means we have to add phpunit-bridge to phpunit's composer.json, no choice